_config.yml

```diff
@@ -2,25 +2,25 @@
 #
 # Use of `relative_permalinks` ensures post links from the index work properly.
 permalink:           pretty
-
-
 # Setup
 title:               Artificial Intelligence
 tagline:             'AIMA Exercises '
 description:         'AIMA Exercises: Exercises for the book Artificial Intelligence: A Modern Approach'
 url:                 https://aimacode.github.io/aima-exercises/
 baseurl:             '/aima-exercises'
 incremental: true
-
-
 # About/contact
 author:
   name:              Sachin Chopra
   url:               sachin10101998.github.io
   email:             schopra@mt.iitr.ac.in
-
 # Custom vars
 version:             1.0.0
 plugins:
   - jekyll-paginate
```

Comment:
Have you checked if jekyll-paginate works with gh-pages or did you pushed it only after checking on your local jekyll server? As far as I know, gh-pages **_does not_** support jekyll-paginate. Also, I don't think we discussed the use of this plugin in any of our status meetings either. So any particular reason to include this?

Reply:
Removed pagination as it wasn't needed in our project.